cmd/pets/dry-run.go

```diff
@@ -13,7 +13,7 @@ var DryRunCmd = &cobra.Command{
 	Run: func(cmd *cobra.Command, args []string) {
 		file := mill.GetFilePath()
 
-		err := mill.ExecFile(file, os.Stdout)
+		err := (*mill.Probe).ExecFile(&mill.Probe{Stdout: os.Stdout, Stderr: os.Stderr}, file)
```

Comment:
aghhh i forgot this syntax even existed.

usually, you would do this as 
probe := &mill.Probe{Stdout: os.Stdout, Stderr: os.Stderr}
err := probe.ExecFile(file)
the syntax you're using is for some rare corner cases and usually not used in idiomatic Go